src/method/method_invoke_java.cpp

```diff
@@ -200,6 +209,15 @@ namespace cubmethod
 	value_unpacker.value = &temp;
 	value_unpacker.unpack (unpacker);
 
+	if (db_value_type (&returnval) == DB_TYPE_RESULTSET)
+	  {
+	    std::uint64_t query_id = db_get_resultset (&returnval);
+	    query_cursor *cursor = m_cursor_map [query_id];
+	    cursor->close ();
+	    delete cursor;
+	    m_cursor_map.erase (query_id);
+	  }
+
```

Comment:
This routine has already been checked in the preceding code. It looks redundant.

Reply:
I've fixed